### PR TITLE
Blito's First Goodie

### PR DESCRIPTION
**Pull request by Blito**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Says I'm awesome.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It teaches ignorant people about how awesome I am.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Myself. You can know more about me once you merge this pull request and look for "duckduckhack blito" in duckduckgo.com

**Why did you choose this data source?**
It's the best one available.

**Are there any other alternative (better) data sources?**
Better? Not sure.

**What are some example queries that trigger this Instant Answer?**
duckduckhack blito

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
The guys at the office! And my girlfriend too, probably.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
It's a tutorial.

**Which existing Instant Answers will this one supercede/overlap with?**
I'm pretty sure there are other awesome people who did this too.

**Are you having any problems? Do you need our help with anything?**
No, thanks.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Duckduckgo's homepage.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
tutorial

## Checklist
Please place an 'X' where appropriate.

```
[X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [X] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

File: t/IsAwesome/Blito.t [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+use warnings;
+use Test::More;
+use DDG::Test::Goodie;
+
+zci answer_type => "is_awesome_blito";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    [qw( DDG::Goodie::IsAwesome::Blito )],
+    # At a minimum, be sure to include tests for all:
```
Comment (mintsoft) [inline t/IsAwesome/Blito.t]: You can delete these comments, they're only for instruction :)

**Comment by mintsoft** [comment]
@Blito Congrats on your first PR! It looks ready to merge once those comments have been deleted and I've seen the tests are green :+1:

**Comment by Blito** [comment]
Done! Thanks for replying so quickly!

**Comment by mintsoft** [comment]
Excellent; congrats on your first Goodie! :cake:

**Comment by moollaza** [comment]
@Blito it's live! https://duckduckgo.com/?q=+duckduckhack+blito&ia=answer

Now that you've completed your first Goodie, we really hope you'll work on another! Feel free to check out our [Ideas Forum](https://duck.co/ideas) or look through our Github issues if you need something to work on!

**Pull request merged**